dhis-2/dhis-services/dhis-service-core/src/main/java/org/hisp/dhis/dataelement/DefaultCategoryService.java

```diff
@@ -883,4 +883,32 @@ public void deleteCategoryOptionGroupSet( CategoryOptionGroupSet group )
     {
         return categoryOptionGroupSetStore.getCategoryOptionGroupSetsNoAcl( DataDimensionType.ATTRIBUTE, true );
     }
+
+    @Override
+    @Transactional( readOnly = true )
+    public Category getDefaultCaEagerFetch()
+    {
+        return categoryStore.getDefaultCat();
+    }

```

Comment:
Move various methods under the sections where they belong, i.e. under each object type individually.

Reply:
Same comment as above regarding non standard semantic